lib/aws_loader.rb

```diff
@@ -1,9 +1,13 @@
 require 'yaml'
-require_relative '../config/s3_store'
 module AwsLoader
   
-  AWS_PATH = File.join(QIt.root, 'config/aws.yml')
+  AWS_PATH = if File.directory?(Application.root + '/config')
+               File.join(Application.root, 'config/aws.yml') 
+             else
+               FileUtils.mkdir_p(Application.root + '/config')[0]
+             end
```

Comment:
L4 checks if the `config` folder exists - and not the existence of the `aws.yml` file. The code unexpectedly exits - as well as the rspec breaks when there is no `aws.yml` present but the `config` folder exists.

Reply:
Making changes to this file itself.